src/gstreamer/gstkvssink.cpp

```diff
@@ -423,6 +435,13 @@ void create_kinesis_video_stream(GstKvsSink *kvssink) {
             break;
     }
 
+    // StreamDefinition takes in C++ strings, check the gchar* for nullptr before constructing
+    if (kvssink->stream_name == NULL || kvssink->content_type == NULL ||
+        kvssink->user_agent == NULL || kvssink->kms_key_id == NULL ||
+        kvssink->codec_id == NULL || kvssink->track_name == NULL) {
+        throw runtime_error(string(INTERNAL_CHECK_PREFIX) + " A string was unexpectedly NULL!");
```

Comment:
Please add more context to this log such as "A StreamDefinition string member was..."

Reply:
I extracted the logic to a macro to print out the exact member that was null.

Reply:
Checked locally:
```
0:00:00.039928791  3604 0x6000011501e0 WARN                 kvssink gstkvssink.cpp:1667:gst_kvs_sink_change_state:<kvssink> error: Failed to init kvs producer. Error: Assertion failed: kvssink->content_type is unexpectedly NULL!
```